test/smoke/faucet.js

```diff
@@ -1,4 +1,27 @@
+const SSH = require('simple-ssh');
+const fs = require('fs');
+const utils = require('../testUtils');
+
+const getNetworkConfig = require('../../lib/test/getNetworkConfig');
+
+const networkConfig = getNetworkConfig();
+
+
 describe('Faucet', () => {
-  // getFunds
-  it('should respond positive balance');
+  networkConfig.inventory.masternodes.hosts.forEach((nodeName) => {
+    it('should respond positive balance via insight and cli',
+      async () => {
+        const response = await utils.requestWrapper(`http://${networkConfig.inventory._meta.hostvars[nodeName].public_ip}:3001/insight-api-dash/addr/${networkConfig.variables.faucet_address}`);
```

Comment:
I was thinking we just call faucet service on 80 port. `getFunds` RPC call.

Reply:
we were not able to get faucet via rpc.getFunds. only check balance by direct IP:80/ request